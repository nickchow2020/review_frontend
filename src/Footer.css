.footerWrapper{
  height: 15rem;
}

.footerWrapper h3{
  margin-top: 2rem;
}