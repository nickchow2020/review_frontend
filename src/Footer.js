import React from "react"
import "./Footer.css"


const Footer = ()=>{
  return (
    <div className="footerWrapper">
        <h3>Copyright Â© 2021 Shumin Inc.</h3>
    </div>
  )
}

export default Footer