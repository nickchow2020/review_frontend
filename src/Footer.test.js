import React from 'react';
import { render } from '@testing-library/react';
import Footer from "./Footer";

it ("should render", function(){

  render(<Footer />)
})

it ("should match snapshot", function(){
  const {asFragment} = render(<Footer />)
  expect(asFragment()).toMatchSnapshot()
})