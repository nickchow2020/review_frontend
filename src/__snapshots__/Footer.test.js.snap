// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot 1`] = `
<DocumentFragment>
  <div
    class="footerWrapper"
  >
    <h3>
      Copyright Â© 2021 Shumin Inc.
    </h3>
  </div>
</DocumentFragment>
`;
